### PR TITLE
use atom-jasmine3-test-runner

### PR DESCRIPTION
**Pull request by UziTech**

### CODE REVIEW

**Review by aminya — APPROVED** [review]
Awesome!

**Comment by UziTech** [comment]
Should we merge this before #153 and see if it fixes the tests or should I rebase this after merging that?

**Comment by aminya** [comment]
I think we can merge this. Rebasing would be a pain, but this PR might fix Ubuntu/MacOS crashes.

**Pull request merged**
